dGame/TradingManager.cpp

```diff
@@ -103,6 +103,7 @@ void Trade::SetAccepted(LWOOBJID participant, bool value) {
 		}
 
 		Complete();
+		TradingManager::Instance()->CancelTrade(m_TradeId);
```

Comment:
Why is this here?

Reply:
Because previously this was at the bottom of complete, however we may not always hit that.